packages/commonwealth/client/scripts/views/pages/user_dashboard/TrendingCommunitiesPreview/TrendingCommunitiesPreview.tsx

```diff
@@ -10,6 +10,12 @@ export const TrendingCommunitiesPreview = () => {
 
   const sortedCommunities = app.config.chains
     .getAll()
+    .filter((community) => {
+      const name = community.name.toLowerCase();
+      return (
```

Comment:
please add the comment why we do this as without the context it looks odd

Reply:
@masvelio Just added a comment. Good call.